scripts/scraper-ng/plugins/identify-recipes.js

```diff
@@ -0,0 +1,86 @@
+const path = require("path");
+
+const signatures = [
+  {
+    recipePath: recipe("javascript-class"),
+    conditions: {
+      tags: ["JavaScript", "Class"]
+    }
+  },
+  {
+    recipePath: recipe("javascript-constructor"),
+    conditions: {
+      tags: ["JavaScript", "Constructor"]
+    }
+  },
+  {
+    recipePath: recipe("javascript-error"),
+    conditions: {
+      tags: ["JavaScript", "Error", "Constructor"]
```

Comment:
I should have seen this before, but why should these tags map to "javascript-error"? I would have expected `["JavaScript", "Errors"]` to catch pages like https://wiki.developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Malformed_formal_parameter.

Reply:
Because I mistakenly thought the javascript-error recipe meant `Error`-like objects. I just pushed a limited fix.

That said, I don't want to get too stuck on the tag set matching before I've written matching for all the JavaScript recipes. It's not yet clear to me what will constitute an implementation problem with this script and what will constitute an upstream content issue. Just for starters, the set of tags `["JavaScript", "Errors"]` won't do the trick for pages under `/en-US/docs/Web/JavaScript/Reference/Errors`: https://gist.github.com/ddbeck/3253e48f2643cd23b6cd0aa978482fe2

Reply:
> It's not yet clear to me what will constitute an implementation problem with this script and what will constitute an upstream content issue. 

Yes, that's a fair point. I just looked at one `Error` page and it had those tags. In fact `Error` seems like a better tag than `Errors` and it's quite possible that the pages use a mixture...